src/PL/Password/Generator/Option.java

```diff
@@ -0,0 +1,151 @@
+package PL.Password.Generator;
+
+import java.awt.*;
+import java.awt.datatransfer.Clipboard;
+import java.awt.datatransfer.StringSelection;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.Scanner;
+
+import static PL.Password.Generator.CommonlyMethods.clearConsole;
+import static PL.Password.Generator.CommonlyMethods.isaBoolean1to2;
+import static PL.Password.Generator.Program.start;
+import static PL.Password.Generator.Tab.*;
+
+
+public class Option {
+
+    private static int countTab = 4;
+
+    public static int getCountTab() {
+        return countTab;
+    }
+
+    public static String little() {
+
+        return isActivateLittle() ? "1. [*] little" : "1. [ ] little";
+    }
+
+    public static String big() {
+        return isActivateBig() ? "2. [*] big" : "2. [ ] big";
+    }
+
+    public static String number() {
+        return isActivateNumber() ? "3. [*] number" : "2. [ ] number";
+    }
+
+    public static String symbol() {
+        return isActivateSymbol() ? "4. [*] symbol" : "2. [ ] symbol";
+    }
+
+
+    public static void changeStatus(int choose) {
+
+        if (choose == 1) {
+            setActivateLittle(!isActivateLittle());
+            if (isActivateLittle()) {
+                countTab++;
+            } else countTab--;
+        } else if (choose == 2) {
+            setActivateBig(!isActivateBig());
+            if (isActivateBig()) {
+                countTab++;
+            } else countTab--;
+        } else if (choose == 3) {
+            setActivateNumber(!isActivateNumber());
+            if (isActivateNumber()) {
+                countTab++;
+            } else countTab--;
+        } else if (choose == 4) {
+            setActivateSymbol(!isActivateSymbol());
+            if (isActivateSymbol()) {
+                countTab++;
+            } else countTab--;
+        }
+
+        start();
+    }
+
+
+    public static void generator() {
+
+        if (countTab == 0) {
+            Tab.setPassword("None of the options have been selected");
```

Comment:
`Tab.setPassword()` This looks like a hack with the message you're passing here :-D